src/plugins/controls/public/control_types/options_list/options_list_editor_options.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+import { EuiFormRow, EuiSwitch } from '@elastic/eui';
+
+import { OptionsListEmbeddableInput } from './types';
+import { OptionsListStrings } from './options_list_strings';
+import { ControlEditorProps } from '../..';
+
+interface OptionsListEditorState {
+  singleSelect?: boolean;
+  runPastTimeout?: boolean;
+}
+
+export const OptionsListEditorOptions = ({
```

Comment:
This is great! I was wondering why the diff actually _removed_ lines of code, this is much more efficient! :fire::fire::fire: